cms/static/js/views/pages/container.js

```diff
@@ -293,13 +293,29 @@ function($, _, Backbone, gettext, BasePage, ViewUtils, ContainerView, XBlockView
                     }));
                 }
                 if (newFiles.length) {
-                    notices.push(() => new NotificationView.Confirmation({
-                        title: gettext("New files were added to this course's Files & Uploads"),
+                    notices.push(() => new NotificationView.Info({
+                        title: gettext("New file(s) added to Files & Uploads."),
                         message: (
-                            gettext("The following required files were imported to this course:") +
+                            newFiles.length + " " + gettext("new file(s) added to your course:") +
```

Comment:
^ The above will be too hard to translate, because in other languages the number may not always come at the beginning of the string, and many languages need to use different inflection based on how many files there are. You can solve this by using `ngettext` instead (search for examples in the codebase already), or the phrase I had which didn't specify the number of files.

(Same in `course_outline.js` too.)

Reply:
Got it, I went with the simpler approach of using the existing phrase that doesn't specify any numbers in it.